header-verifier/tests/header_verifier_blackbox_tests.rs

```diff
@@ -588,7 +589,7 @@ fn test_change_validator_set() {
 /// H-VERIFIER_CHANGE_VALIDATORS_FAIL
 ///
 /// ### ACTION
-/// Call 'change_validators_set()' after registering the operation
+/// Call 'change_validator_set()' before registering the operation
```

Comment:
I see this test is kind of bad...
We need to do `change_validator_set` 2 times and get `OUTGOING_TX_HASH_ALREADY_REGISTERED`,
not `register_operations` and `change_validator_set`.